blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/formats/MetricData.java

```diff
@@ -40,44 +40,14 @@ public String getUnit() {
     }
 
     public String getType() {
```

Comment:
Does anyone still call this? I wonder what that would be for. May be we can get rid of Type completely.